project/build.properties

```diff
@@ -12,4 +12,4 @@
 # limitations under the License.
 #
 
-sbt.version=1.0.2
+sbt.version=1.0.3
```

Comment:
sbt-1.0.2 has a bug in finding resources and log file discovery generated in test case failes in 1.0.2.